README.md

```diff
@@ -114,16 +136,26 @@ enum TaskStatus: int
     case COMPLETED = 1;
     case CANCELED = 2;
 }
+
+enum Role
+{
+    use Values;
+
+    case ADMINISTRATOR;
+    case SUBSCRIBER;
+    case GUEST;
+}
 ```
 
 #### Use the `values()` method
 ```php
 TaskStatus::values(); // [0, 1, 2]
+Role::values(); // ['ADMINISTRATOR', 'SUBSCRIBER', 'GUEST']
 ```
 
 ### Options
 
-This helper returns an associative array of case names and values.
+This helper returns an associative array of case names and values for backed enums, or a list of names for pure enums (making this functionally equivalent to [`::names()`](#names) for pure Enums).
```

Comment:
I'll merge this now, but just want to confirm: for pure enums, we're using names for **everything**, right?

Meaning, names, values, options all return the names. And the internal indices aren't used for anything.